google/cloud/storage/blob.py

```diff
@@ -692,12 +703,21 @@ def exists(
             if_metageneration_match=if_metageneration_match,
             if_metageneration_not_match=if_metageneration_not_match,
         )
+
+        headers = {}
+        _add_etag_match_headers(
+            headers, if_etag_match=if_etag_match, if_etag_not_match=if_etag_not_match
+        )
+        if not headers:
+            headers = None
+
```

Comment:
No harm passing the empty dict here:
```suggestion
```

Reply:
Cool, I didn't know if downstream was tolerant to that. Will remove all of these.